src/main/java/com/github/alexthe666/iceandfire/event/WorldGenDreadDimension.java

```diff
@@ -32,12 +29,11 @@ public void generate(Random random, int chunkX, int chunkZ, World world, IChunkG
             BlockPos height = getHeight(world, new BlockPos(x, 0, z));
 
             if (BiomeDictionary.hasType(world.getBiome(height), BiomeDictionary.Type.COLD) && BiomeDictionary.hasType(world.getBiome(height), BiomeDictionary.Type.SNOWY)) {
-                if (random.nextInt(1000) == 0) {
```

Comment:
What is the purpose for removing this? (I think I know why but I'm double-checking)

Reply:
because now castle will spawn each 2000 block instead of spawn with randomly